crates/pessimistic-proof/src/imported_bridge_exit.rs

```diff
@@ -0,0 +1,57 @@
+use serde::{Deserialize, Serialize};
+
+use crate::{
+    bridge_exit::{BridgeExit, NetworkId},
+    keccak::{Digest as KeccakDigest, Digest},
+    local_exit_tree::{data::LETMerkleProof, hasher::Keccak256Hasher},
+};
+
+/// Represents a token bridge exit originating on another network but claimed on the current network.
+#[derive(Debug, Clone, Serialize, Deserialize)]
+pub struct ImportedBridgeExit {
+    /// The bridge exit initiated on another network, called the "foreign" network.
+    /// Need to verify that the destination network matches the current network, and that
+    /// the bridge exit is included in an imported LER
+    pub bridge_exit: BridgeExit,
+
+    /// The Network ID for the foreign network. May not be strictly
+    /// necessary depending on the final structure of the Imported Local Exit Root
+    pub sending_network: NetworkId,
```

Comment:
nit: referred as `foreign network` in the rustdoc also above for `bridge_exit`, but the variable is named as `sending_network`, any of the two names is ok with me

Reply:
I find "sending" more explicit. Will change in the doc comments 👍